aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/jaxrs/AwsProxySecurityContext.java

```diff
@@ -97,14 +92,12 @@ public Principal getUserPrincipal() {
                         return event.getRequestContext().getIdentity().getUserArn();
                     }
                 }
-
-                // return null if we couldn't find a valid scheme
                 return null;
             };
         }
 
         if (getAuthenticationScheme().equals(AUTH_SCHEME_COGNITO_POOL)) {
-            return new CognitoUserPoolPrincipal(event.getRequestContext().getAuthorizer().getClaims());
+            return new CognitoUserPoolPrincipal((Map<String, String>) event.getRequestContext().getAuthorizer().get("claims"));
```

Comment:
Please define a constant (ideally it should be provided by aws-lambda-events lib).

Reply:
Updated